recipes/tl-optional/all/conandata.yml

```diff
@@ -0,0 +1,7 @@
+sources:
+  "1.0.0":
+    url: https://github.com/TartanLlama/optional/archive/v1.0.0.zip
+    sha256: 8bb68defc61da3de2b4cd73ef9792eba44570ec5cb52c4da731286f24aecbb95
+  "20191111":
```

Comment:
If you need something from the 12 commits since release 1.0.0 please ask first for a new release upstream

If the answer is no or you don't get an answer, you can add a `cci.<date-of-commit-YYYYMMDD>` version

Reply:
Giving it a look, it doesn't look like there is any significant changes. So I've removed the commit version.

Sorry that I missed a file the first time, had to create another commit (without force pushing)